debian/changelog

```diff
@@ -1,3 +1,39 @@
+snapcraft (2.32) xenial; urgency=medium
+
+  [ Michał Sawicz ]
+  * autotools: don't force uniqueness on configflags (#1337)
```

Comment:
autotools plugin, to be consistent with the others.
I need to get better at catching these before merge.